docker/docker-files/pgCreateTable.sql

```diff
@@ -1,10 +1,30 @@
 DROP TABLE IF EXISTS user_data;
--- key,id,data
+
 CREATE TABLE user_data (
     docKey VARCHAR(255) NOT NULL,
-    docType VARCHAR(255) NOT NULL,
+    docZone VARCHAR(255) NOT NULL,
+    docOwner VARCHAR(255) NOT NULL,
+    docOrganization VARCHAR(255) NOT NULL,
     id VARCHAR(255) PRIMARY KEY,
-    creationDate TIMESTAMP,
+    lastUpdateDate TIMESTAMP,
     docValue JSON
 );
-CREATE INDEX doc_key_idx ON user_data (docKey, docType);
\ No newline at end of file
+CREATE UNIQUE INDEX doc_key_idx_orga ON user_data (docKey, docZone,docOrganization);
+
+DROP TABLE IF EXISTS user_data_readers;
+
+CREATE TABLE user_data_readers (
+    data_id VARCHAR(255) ,
+    readers VARCHAR(255) NOT NULL,
+    FOREIGN KEY (data_id) REFERENCES user_data (id)
+);
+
+DROP TABLE IF EXISTS user_data_writers;
+
+CREATE TABLE user_data_writers (
+    data_id VARCHAR(255) ,
+    writers VARCHAR(255) NOT NULL,
```

Comment:
is it one writer or several writers?

Reply:
This table contains one line by couple data_id / group allowed to write; so i remove the 's' in the name of the table and in the name of the column ?

Reply:
I would remove the 's' from the column name